src/components/backend-ai-storage-list.ts

```diff
@@ -928,11 +930,12 @@ export default class BackendAiStorageList extends BackendAIPage {
     } else {
       this.shadowRoot.querySelector('vaadin-grid.folderlist').style.height = 'calc(100vh - 185px)';
     }
-    document.addEventListener('backend-ai-group-changed', (e) => this._refreshFolderList());
+    document.addEventListener('backend-ai-group-changed', (e) => this._refreshFolderList(true, 'group-changed'));
     document.addEventListener('backend-ai-ui-changed', (e) => this._refreshFolderUI(e));
     this._refreshFolderUI({'detail': {'mini-ui': globalThis.mini_ui}});
     // monkeypatch for height calculation.
     this.selectAreaHeight = this.shadowRoot.querySelector('#dropdown-area').offsetHeight ? this.shadowRoot.querySelector('#dropdown-area').offsetHeight : '56px';
+    this._triggerFolderListChanged();
```

Comment:
I added this function [to initialize the storage status chart graph](https://github.com/lablup/backend.ai-webui/blob/554680e3bb85b4023f485ea9ff4420ad2ed3eddb/src/components/backend-ai-data-view.ts#L544-L547).